ui/src/components/SchemaTable/index.jsx

```diff
@@ -119,14 +107,18 @@ export default class SchemaTable extends Component {
     }
   }
 
-  readReference(schemaPath) {
-    const { serviceName } = this.props;
-    const id = schemaPath.startsWith('/')
-      ? schemaPath
-      : join('/', 'schemas', serviceName, schemaPath);
-    const schemaId = id.endsWith('#') ? id : `${id}#`;
+  readReference(schemaId) {
+    const fullSchemaId =
+      schemaId.endsWith('#') ? schemaId : `${schemaId}#`;
```

Comment:
Linter is not happy here.

Reply:
Lol, of course, I change a few characters and the linter hates me :)